CRITICAL - 2022-04-28 08:02:55 --> Cannot use object of type CodeIgniter\Files\File as array
#0 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#1 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-04-28 09:03:54 --> There is no data to insert.
#0 C:\xampp\htdocs\UF2-ACTFinal\system\BaseModel.php(718): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
#1 C:\xampp\htdocs\UF2-ACTFinal\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#2 C:\xampp\htdocs\UF2-ACTFinal\system\BaseModel.php(658): CodeIgniter\Model->insert(Object(CodeIgniter\Files\File), false)
#3 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(74): CodeIgniter\BaseModel->save(Object(CodeIgniter\Files\File))
#4 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#5 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#6 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-04-28 09:09:58 --> Cannot use object of type CodeIgniter\Files\File as array
#0 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#1 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-04-28 09:16:26 --> Undefined variable: uploaded_flleinfo
#0 C:\xampp\htdocs\UF2-ACTFinal\app\Views\app\succes.php(13): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 13, Array)
#1 C:\xampp\htdocs\UF2-ACTFinal\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\UF2-ACTFinal\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\UF2-ACTFinal\system\Common.php(1089): CodeIgniter\View\View->render('app/succes', Array, true)
#4 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(75): view('app/succes', Array)
#5 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#6 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#7 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-04-28 09:28:49 --> syntax error, unexpected ']'
#0 C:\xampp\htdocs\UF2-ACTFinal\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\UF2-ACTFinal\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-04-28 09:29:05 --> Argument 2 passed to view() must be of the type array, object given, called in C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php on line 76
#0 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(76): view('app/succes', Object(CodeIgniter\Files\File))
#1 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#3 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-28 09:33:01 --> file_get_contents(C:\xampp\tmp\phpBB3A.tmp): failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp\\htdocs...', 72, Array)
#1 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(72): file_get_contents('C:\\xampp\\tmp\\ph...')
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#3 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#4 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-28 09:33:24 --> file_get_contents(C:\xampp\tmp\php1707.tmp): failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp\\htdocs...', 72, Array)
#1 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(72): file_get_contents('C:\\xampp\\tmp\\ph...')
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#3 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#4 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-28 09:35:26 --> Undefined variable: writepath
#0 C:\xampp\htdocs\UF2-ACTFinal\app\Views\app\succes.php(18): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 18, Array)
#1 C:\xampp\htdocs\UF2-ACTFinal\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\UF2-ACTFinal\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\UF2-ACTFinal\system\Common.php(1089): CodeIgniter\View\View->render('app/succes', Array, true)
#4 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(78): view('app/succes', Array)
#5 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#6 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#7 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-04-28 10:39:15 --> file_get_contents(C:\xampp\tmp\php62B9.tmp): failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp\\htdocs...', 64, Array)
#1 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(64): file_get_contents('C:\\xampp\\tmp\\ph...')
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#3 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#4 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-28 10:41:11 --> Undefined variable: imgData
#0 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(73): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 73, Array)
#1 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#3 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-28 10:42:23 --> file_get_contents(C:\xampp\tmp\php414C.tmp): failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_get_conten...', 'C:\\xampp\\htdocs...', 68, Array)
#1 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(68): file_get_contents('C:\\xampp\\tmp\\ph...')
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#3 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#4 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-28 10:48:36 --> mysqli_connect(): (HY000/1049): Unknown database 'gndaw2022'
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', 'C:\\xampp\\htdocs...', 39, Array)
#1 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(39): mysqli_connect('localhost', 'root', '', 'gndaw2022')
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#3 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#4 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-28 11:07:16 --> Undefined property: CodeIgniter\HTTP\Files\UploadedFile::$type
#0 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(72): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 72, Array)
#1 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#3 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-28 11:42:36 --> mysqli_connect(): (HY000/1049): Unknown database 'gndaw2022'
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli_connect(...', 'C:\\xampp\\htdocs...', 39, Array)
#1 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(39): mysqli_connect('localhost', 'root', '', 'gndaw2022')
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#3 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#4 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-04-28 11:43:48 --> Undefined variable: db
#0 C:\xampp\htdocs\UF2-ACTFinal\app\Controllers\Dashboard.php(79): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 79, Array)
#1 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#3 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-04-28 11:44:21 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::insert()
#0 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(832): App\Controllers\Dashboard->upload()
#1 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#2 C:\xampp\htdocs\UF2-ACTFinal\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\UF2-ACTFinal\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
